CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+We'd love some help making this wiki! Get started by creating a fork, commiting some changes, then creating a pull request!
+
+## Format
+```
+# Title
+[Short introduction]
+## Topic
+[Contents]
+## Topic
+[Contents]
+## Example (optional)
+[Example]
+```
+
+If you don't have time or can't finish a wiki page, put this block at the end:
```

Comment:
Would it be better if it was at the top of the page? People would be more likely to notice and thus help?

Reply:
I added the tips in a the order I remembered them, so yes, it would definitely make more sense there.